components/automate-chef-io/content/docs/datafeed.md

```diff
@@ -0,0 +1,122 @@
++++
+title = "Data Feeds"
+description = "Exporting Node Data from Chef Client Runs to External Data Collectors"
+date = 2020-05-05T13:19:02-07:00
+draft = false
+bref = ""
+toc = true
+[menu]
+  [menu.docs]
+    parent = "settings"
+    weight = 10
++++
+
+## About Automate Data Feeds
+
+The Data Feed service is used to send node data to a 3rd party service. This can be useful when updating Configuration Management Databases, external Security Dashboards and ITSM platforms.
+The following types of information are sent:
+ 
+ - Ohai data gathered from each node being managed, this includes HW, OS and installed program information, there is some variation depending on the OS being managed
+ - Configuration information about each node being managed, this includes Chef Client Run status, Runlists, Cookbooks, Recipes being run against each node
+ - Compliance information about each node that shows the compliance state, this includes passed and failed controls for each profile executed against that node
+
+A Data Feed is not a Data Tap, it operates by doing the following:
+
+- Every 4 hours (configurable) the data-feed-service will aggregate the client runs and compliance reports from the previous 4 hours and send this to the registered destinations.
+- If there are no destinations, aggregation will not occur.
+- The data is aggregated and sent in batches of 50 nodes at a time (also configurable)
+
+
+By default only Admins of Chef Automate may create and manage Data Feeds.
+
+### Adding a Data Feed Endpoint
```

Comment:
@davymcaleer Are there any restrictions to setting up more than one Data Feed endpoint? Asking just in case! :)

Reply:
See line 64 to highlight this

Reply:
I modified line 64 to be an info box (lines 65-67) highlighting the global application of the `config.toml` file.